### PR TITLE
Demo Chatbot in plugins 

### PR DESCRIPTION
**Pull request by dereksmart**
### A demo branch for the Jetpack Chatbot package and Odie. 
This PR is holding the running commits of the new (and experimental) [Jetpack Chatbot](https://github.com/Automattic/jetpack-chatbot) package as an easier way for others to test it on live sites. It's not meant to be merged. The good changes will be merged to the package directly in separate PRs, while this branch is kept up to date. 

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

TL;DR
- Set up odie-client per PCYsg-Skx-p2
- In the beta plugin settings, choose this branch for Jetpack or Starter Plugin and activate it. 
- Connect the site to your wpcom user. 
- Sandbox the site's connection. 
- Behold.  

Detailed instructions can be found in 218-gh-Automattic/ai-services

### CODE REVIEW

File: .github/files/list-changed-projects.sh [hunk 1]
```diff
@@ -15,6 +15,22 @@ function die {
 	exit 1
 }
 
+# Define specific projects.
+SPECIFIC_PROJECTS=(
```
Comment (dereksmart) [inline .github/files/list-changed-projects.sh:19]: Custom build rules so it doesn't take 23 minutes every commit
